static/js/tools/choropleth/choropleth_template.tsx

```diff
@@ -155,9 +154,9 @@ class MainPane extends Component {
         </div>
         <div id="main-content">
           <React.Fragment>
+          <div className="column" id="breadcrumbs"></div>
             <div id="heading">Loading...</div>
             <div>
-              <div className="column" id="breadcrumbs"></div>
               <div className="column" id="hover-text-display"></div>
             </div>
```

Comment:
I suspect it's because you're using columns, but without adding rows.
https://getbootstrap.com/docs/4.0/layout/grid/

Did you try removing column from the divs? Or using something like this:
<div classname=row><div classname=col-12 id=breadcrumbs></div></div>

Reply:
Not sure what you mean here.

Reply:
it's ok. i wasn't sure what the column classes were doing, or if it was meant to be the bootstrap grid classes.